sunpy/net/__init__.py

```diff
@@ -2,4 +2,14 @@
 
 __all__ = ["Fido"]
 
+
+# Import and register the clients but we do not want them in the namespace, we import them as _
+# Add the JSOC and VSO Clients explicitly as they do not inherit from {Base,Generic}Client
+from sunpy.net import base_client as _base_client
+from sunpy.net.jsoc.jsoc import JSOCClient as _
+_base_client.BaseClient._registry[_] = _._can_handle_query
```

Comment:
We don't need these now right? As the registration is under `BaseClient` rather than `GenericClient`